explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+Thing Description is not limited to http based interfaces as shown in the example above. In general, WoT is a protocol agnostic approach and provides a common mechanism to define how specific protocols such as MQTT, HTTP, CoAP or Modbus can be mapped to the WoT’s interaction properties-action-event abstraction within the Thing Description forms definition. More details is provide in the [Protocol Binding section](https://w3c.github.io/wot-thing-description/#protocol-bindings).  
+ 
+
+### TD Context Extensions
+Through JSON-LD serialization, the WoT Thing Description provides the ability to add context knowledge from additional namespaces. This mechanism can be used to enrich the Thing Description instances with additional (e.g., domain-specific) semantics. It can also be used to import additional Protocol Bindings or new security schemes in the future. 
+
+ The following TD extends the TD above by introducing a second definition in the @context to declare the prefix saref as referring to SAREF. This IoT ontology includes terms interpreted as semantic labels that can be set as values of the @type field, giving the semantics of Things and their interaction affordances. In the example below, the Thing is labelled with saref:LightSwitch, the status Property is labelled with saref:OnOffState and the toggle Action with saref:ToggleCommand.
+
+ ```json
+{
+    "@context": [
+        "https://www.w3.org/2022/wot/td/v1.1",
+        { "saref": "https://w3id.org/saref#" }
+    ],
+    "title": "MyLampThing",
+    "@type": "saref:LightSwitch",
+    "securityDefinitions": {
+        "basic_sc": {"scheme": "basic", "in": "header"}
+    },
+    "security": "basic_sc",
+    "properties": {
+        "status": {
+            "@type": "saref:OnOffState",
+            "type": "string",
+            "forms": [{
+                "href": "https://192.168.2.134/status"
+            }]
+        }
+    },
+    "actions": {
+        "toggle": {
+            "@type": "saref:ToggleCommand",
+            "forms": [{
+                "href": "https://192.168.2.134/toggle"
+            }]
+        }
+    },
+    "events": {
+        "overheating": {
+            "data": {"type": "string"},
+            "forms": [{
+                "href": "https://192.168.2.134/oh"
+            }]
+        }
+    }
+}
+```
+### Thing Model definition
+One of the main intentions of a Thing Description is to provide a Consumer with all the details necessary to successfully interact with a Thing. In some IoT application scenarios, a fully detailed Thing Description, e.g., with communication metadata is not necessary (e.g., IoT ecosystems may implicitly handle communication separately), or may not be available because a new entity has not yet been deployed (e.g., IP address is not yet known). Sometimes, also a kind of class definition is required that forces capability definitions that should be available for all created instances (e.g., large-scale production of new devices).
```

Comment:
this is the first place you use "Consumer", but without a definition.  I would say "client" here to be consistent, OR (much earlier) talk about kinds of clients (developers reading the spec, Consumer device/services. etc.).

Reply:
e.g. some IoT ecosystems may...

"forces" -> "provides"

Reply:
done